src/core/hooks/use-form-handle/use-form-handle.ts

```diff
@@ -64,6 +67,18 @@ export const useFormHandle = <Values extends object, Err>(
           state.failedSubmitCount,
           state.touched
         ),
+        Atom.fuse(
+          (...fieldsChanged) => fieldsChanged.some(Boolean),
+          ...values(Schema).map(field => {
+            const fieldLens = impl(field).__lens;
+            const initialValue = fieldLens.get(state.initialValues);
+            const fieldAtom = Atom.entangle(state.values, fieldLens);
+            return Atom.fuse(
+              fieldValue => !deepEqual(fieldValue, initialValue),
+              fieldAtom
+            );
+          })
+        ),
```

Comment:
wouldn't `deepEqual` on `state.values` and `state.initialValues` be enough? 
```suggestion
        Atom.fuse(values => !deepEqual(values, state.initialValues), state.values),
```

Reply:
yes, this is a bit of an optimisation - we don't throw the entire form values into deepEqual every time any field changes, but only compare the field that did change. If the entire from is nested in object field then it does not matter, but I thought it could be worth it in some cases.